rockcraft/oci.py

```diff
@@ -119,39 +119,58 @@ def copy_to(self, image_name: str, *, image_dir: Path) -> "Image":
 
         return Image(image_name=image_name, path=image_dir)
 
-    def extract_to(self, bundle_dir: Path) -> Path:
+    def extract_to(self, bundle_dir: Path, *, rootless=False) -> Path:
```

Comment:
```suggestion
    def extract_to(self, bundle_dir: Path, *, rootless: bool = False) -> Path:
```

Reply:
I thought it was weird that mypy didn't catch this, so I added #159 . Thanks for noticing this